src/components/datepicker/datePicker.js

```diff
@@ -570,11 +574,10 @@
   /** Close the floating calendar pane. */
   DatePickerCtrl.prototype.closeCalendarPane = function() {
     if (this.isCalendarOpen) {
-      this.isCalendarOpen = false;
       this.detachCalendarPane();
+      this.isCalendarOpen = false;
```

Comment:
I can understand your thoughts. I had the behavior, you desired, before, but I thought calling `enableScrolling` is to generic, because `detachPane` will be always called after `hidePane`, so it made sense to me, to move the enable scroll function completely into the `detachPane` function.